README.md

```
@@ -105,23 +105,23 @@ $ npm run build
 $ npm run cli-dev --workspace=apps/server-core -- start
 ```
 
-It serves the admin console at `http://localhost:3001/admin` from that package's built `dist/`.
+It serves the admin console at `http://localhost:3001/console/admin` from that package's built `dist/`.
 
 **4**. To work on the admin console itself, start its dev server in a second terminal
 
 ```shell
 $ VITE_API_URL=http://localhost:3001 npm run dev --workspace=apps/client-admin-console
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)